sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/functions/FunctionsTest.java

```diff
@@ -38,6 +38,40 @@ public void issue1180() {
         );
     }
 
+    // this is an edge case for negative integer modulo
+    // Rust's default modulo operator currently panics on
+    // INT::MIN / -1 instead of just returning 0
```

Comment:
Rust panics for division, but this is modulo. What is the relationship?

Reply:
`/` is a typo, it should be `%`